### DISCUSSION TITLE
读《置身事内》 | /posts/lxh/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#591] [Announcements]
尽信书，不如无书。

https://yuanfan.rbind.io/posts/lxh/

**Comment by XiangyunHuang** [comment]
你写的不比小Lin 说栏目被拿下的那期差，加油 👏。

纵观历史上，为了消除地区差异、城乡差异和脑力、体力劳动差异，也就是说社会主义不能有资本主义的贫富分化。在建国后，毛主席提出大跃进，人民公社和无产阶级文化大革命。在专制体制下，首先通过领袖崇拜，浮夸，欺上瞒下，洗脑洗嘴，实现差异自动消失，不允许有人提差异问题。实际措施有限制人口流动，建立严格的城乡二元结构，消除城乡差异。青年劳动力上山下乡，在人民公社吃大锅饭，消除地区差异。文革將民族资产阶级和知识分子搞掉，消除体力劳动和脑力劳动的差异（以前除了军队干部，脑力劳动的阶层经济状况很不错的）。

**Reply by earfanfan** [reply]
我发现你写的“纵观历史”这段话有很明显的一种倾向，不过因为三年前我在wuhan2020一个群组里与人讨论问题时，有人说国内国外的左和右是反的以后，我就分不清左右了，所以我看不出来你这是左倾还是右倾。

**Reply by earfanfan** [reply]
换个角度看你这段留言的话，一个人的问题意识决定了看待事物的视角，也就是说一个人的视角会容易受限于自身的问题意识。我看到，你的言语主要是表达了上对下的欺瞒，以及从上到下的某种异化，所以我猜你的问题意识大概就是社会正义的缺失，或者阶层固化，权力集团或利益集团通过权力、利益、资源的再分配剥夺了社会的公共利益，为了掩盖真相，或者说让人们意识不到自身利益被攫取，而对公众进行了铺天盖地的思想上的限制和改造。

**Reply by XiangyunHuang** [reply]
我既不左也不右，只是把事实摆出来。至于你对左还是右的迷惑，我也一直有，从初中开始就迷糊，政治、历史和语文从来没学好过，直到最近才知道它们的含义。许子东在课程《中国现代文学》中给出来了，左代表平等，右代表自由，中代表博爱，我一下子恍然大悟。中国共产党一直以来认为左的是好的，历史上很少有批判左的，只有早期少数时候，比如王明左倾教条主义， 49 年以后毛时代主要是反右，鉴于长期以来的极左，79 年以后邓时代主要是防左。

我其实没明白你后面一段话说的啥，我只是从经济的角度看那段历史的影响，回应你文章中关于经济制度、平等等话题。

**Reply by earfanfan** [reply]
我写后面一段是因为以为你写的和你心里所想的是相反的。看来有可能是我误会你了，在我逐条写出我的理解之前，先吐个槽，左平等和右自由似乎并不是相反的，因为自由和平等字面意义上没撒冲突，但是换成从集权国家的社会秩序角度看，如果说平等约等于绝对秩序，自由约等于绝对失序，这样左右才算是相反的。

>在建国后，毛主席提出大跃进，人民公社和无产阶级文化大革命。

并不是毛主席提出了大跃进，关于大跃进形成的原因，我在周飞舟2014年的《当代中国的中央地方关系》看到这样一段解释。在我最近看的一些书里面，政府意志并不是从上到下一个人的意志，而是在中央和地方之间，甚至在中央-省-市-县-乡五级政府之间也是存在博弈的。

>>1958年2月，在中共中央转发毛泽东的《工作方法六十条（草案）》中，提出了“生产计划两本账”的要求，与这种制度相配套，建立起了“块块为主、条块结合”的计划体制，规定各省市自治区可以对本地区的工农业生产指标进行调整，可以安排本地区的建设投资和人力、财力、物力以及公共事业项目。在这种形势下，“二五”时期国家的工业生产计划中只剩下产品产量指标，“一五”计划中的其他五种指标﻿都被取消了。基建计划中也只管当年的投资和主要建设内容。工业产品中，国家计委统一管理的产品种类也大幅减少。中央直接征收的财政收入比重1959年下降到只占财政总收入的20%。在生产的其他方面，如基建审批权、物资分配权和招工计划权也都全面下放。地方基本可以自主基建投资，中央统配和部管物资从1957年的532种减为1958年429种，到1959年更减少为285种﻿。职工人数和城镇居民数量大幅度增加。这种全面放权一方面迅速刺激了经济增长，另一方面却带来了严重的问题。问题主要表现为严重的“浮夸风”以及经济结构的严重失调，其根源却是伴随着全面放权出现的地方政府指标竞赛行为。

>在专制体制下，首先通过领袖崇拜，浮夸，欺上瞒下，洗脑洗嘴，实现差异自动消失，不允许有人提差异问题。

这句看上去也跟经济没撒关系啊，那段历史非常复杂，可能有很多人去世后直到现在也没能沉冤昭雪，这里不展开，回头我俩可以私下探讨，但确实是看了你下的这些结论让我总是忍不住觉得你是在说反话。

>实际措施有限制人口流动，建立严格的城乡二元结构，消除城乡差异。

确实是户籍制度划分了城市和农村两种户口，也确实近几十年的城乡差异不断扩大可以说是在人口可以流动以后，但限制人口流动跟消除城乡差异真得相离甚远，城乡差异的根源还是在于城市人口集聚效应带来的生产效率远高于农村。限制人口流动这部分也是可以展开写很多，也暂略。

后面那几句的内容也是80年代以前的历史，与94年分税制改革后大兴土地财政有一大段时间上的距离。建国后历史上的财政体制变革对国家和社会造成的重大影响，中央政府每次在财政上的集权和放权都可算是出于经济上进行调整的目的，而不该是强行消除国民思想上对各类差异的认知。

**Comment by XiangyunHuang** [comment]
最近两篇博文感觉有很明确的方向，现当代经济和政治体制，你是带着一个什么样的目的去读这方面的书的？

我先说我自己，我读那一点历史，是因为对当下的事有一些浅浅的疑（困）惑，也突然发现以前所学的历史或多或少失去了历史作为一门学科该有的客观性，因回避掉很多现实问题，失去连贯性和系统性，当然还有内在逻辑性，历史的存在沦为政治工具。以前是死记硬背为了各种考试，直到大学才有一个非智能手机。毕业以后，随着信息的流通，接触到的事，遇到的问题越来越多。又深信历史总是在以不同的面孔重复上演，历史使人明智，所以尝试去重新了解现代历史。套用地理学第一定律来说，离得近的比离得远的更相关，所以，清朝及以前的历史，我是不看的。

**Reply by earfanfan** [reply]
>也突然发现以前所学的历史或多或少失去了历史作为一门学科该有的客观性

这可能是因为我们以前学的历史都是课本、教材上的历史，课本和教材对学生来说本来就应该起到启蒙和灌输认知的作用，应该说是课本这种媒介本身带有政治工具的属性，而历史作为一门学科其实还有许多我们没读到过的知识。了解历史最好不要只去看一个人的解读角度，也不要只看一个学科的解读视角，政治、经济、社会、文化互相之间都是紧密关联的，不同的人会带着不同的问题意识去解读，其实我最近越发觉得自己的视角受限于自己的问题意识和社会阶层。

>又深信历史总是在以不同的面孔重复上演，历史使人明智

历史重复上演这也看是从什么视角去看，如果是从权力争夺、利益倾轧角度看，那确实是的，从古至今都是“天下熙熙，皆为利来，天下攘攘，皆为利往”。不过要是换了很多不同视角看，那也可能是一种螺旋上升结构，或者别的什么。

距离我们最近的就是改革开放近三十年的历史，这些年出的政策很多都对我们的生活影响巨大。不过稍微有点可惜的是，现在不如十几年前开放了。

**Comment by Cloudtq** [comment]
写的细节挺好的，补充了我看书时的困惑
